src/waypoint.jsx

```diff
@@ -333,13 +352,18 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.children) {
+      return <div ref={this.refElement}>{this.props.children}</div>;
```

Comment:
I think that adding a div here will be potentially unexpected for consumers and problematic for styling purposes. For example, if I have some content that I want to be inline-block and I wrap it in a Waypoint, how would I achieve the styling I need?

I [don't think we should allow people to pass in styles or classNames](https://brigade.engineering/don-t-pass-css-classes-between-components-e9f7ab192785) because of how much complexity that adds to people's codebases. 

Instead, we could use`React.Children.only` so the consumer is responsible for adding the wrapping element and styling it how they desire. To make this work, we will also need to add a `refToMeasure` prop, similar to what is outlined here: https://github.com/yannickcr/eslint-plugin-react/issues/678#issuecomment-232293175

What do folks think?